src/control/cmd/dmg/utils_test.go

```diff
@@ -25,54 +25,100 @@ package main
 
 import (
 	"fmt"
+	"strings"
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
 
 	. "github.com/daos-stack/daos/src/control/client"
+	. "github.com/daos-stack/daos/src/control/common"
```

Comment:
What's our style guideline for dot imports? I had stopped doing them because I picked up the idea from some code review or another that we were moving away from it. Might be wrong, though.

Reply:
The community consensus on them is that they're okay-ish in test code, particularly if you're using an import which provides a DSL-like interface for your tests. In production code they're frowned upon because it makes it harder to see at a glance where things are coming from.

Reply:
Works for me.